<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClosingsEditor</class>
 <widget class="QDialog" name="ClosingsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Closings Editor</string>
  </property>
  <widget class="QTableWidget" name="closingsTable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>571</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Closing Information</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPlainTextEdit" name="comments">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="drawerAmount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculated Amount : </string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="closingEndDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Closing End Time : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Closing Start Time : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Drawer Amount :</string>
    </property>
   </widget>
   <widget class="QLabel" name="calculatedAmountLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>_________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comments : </string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="closingStartDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="remainingCommentCharachterLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>_______</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Charachters left:</string>
    </property>
   </widget>
  </widget>
  <widget class="QDateEdit" name="searchStart">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>From : </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="searchEnd">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>To :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>714</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>40</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>330</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>390</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="errLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelEditButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>270</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Edit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
